index.html

```diff
@@ -12,84 +12,76 @@
 </head>
 <body>
   <section id="abstract">
-    <p>This document, “Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)”, describes how the Web Content Accessibility Guidelines (WCAG) version 2.2 [<a href="https://www.w3.org/TR/WCAG22/">WCAG22</a>] principles, guidelines, and success criteria can be applied to Mobile Applications, including but not limited to native mobile apps, mobile web apps and hybrid apps using web components inside native mobile apps. It provides informative guidance (guidance that is not normative and does not set requirements).</p>
-<p>This document is part of a series of technical and educational documents published by the <a href="https://www.w3.org/WAI/">W3C Web Accessibility Initiative (WAI)</a> and available from the <a href="https://www.w3.org/WAI/standards-guidelines/mobile/">Mobile Accessibility Overview</a>.</p>
-
+    <p>This document describes how Web Content Accessibility Guidelines (WCAG) 2.2 [[WCAG22]] principles, guidelines, and success criteria can be applied to mobile applications, including native mobile apps, mobile web apps and hybrid apps using web components inside native mobile apps. It provides informative guidance (guidance that is not normative and does not set requirements).</p>
```

Comment:
I see `[[WCAG22]]` is now a bibliography item, nice. Should also work when generating from markdown?